index.html

```diff
@@ -0,0 +1,968 @@
+          <div>
+            <img src="./images/suit.jpg" alt="image" />
+            <p>
+              <a href="#"
+                >This Mars helicopter will be the first ever to fly on another
+                planet</a
+              >
+            </p>
+            <p class="second-p">
+              <a class="strikethrough" href="#">$299</a> $25
+            </p>
+          </div>
+          <div>
+            <img src="./images/suit.jpg" alt="image" />
+            <p>
+              <a href="#"
+                >This Mars helicopter will be the first ever to fly on another
+                planet</a
+              >
+            </p>
+            <p class="second-p">
+              <a class="strikethrough" href="#">$299</a> $25
+            </p>
+          </div>
+          <div>
+            <img src="./images/suit.jpg" alt="image" />
+            <p>
+              <a href="#"
+                >This Mars helicopter will be the first ever to fly on another
+                planet</a
+              >
+            </p>
+            <p class="second-p">
+              <a class="strikethrough" href="#">$299</a> $25
+            </p>
+          </div>
+          <div>
+            <img src="./images/suit.jpg" alt="image" />
+            <p>
+              <a href="#"
+                >This Mars helicopter will be the first ever to fly on another
+                planet</a
+              >
+            </p>
+            <p class="second-p">
+              <a class="strikethrough" href="#">$299</a> $25
+            </p>
+          </div>
+          <div>
+            <img src="./images/suit.jpg" alt="image" />
+            <p>
+              <a href="#"
+                >This Mars helicopter will be the first ever to fly on another
+                planet</a
+              >
+            </p>
+            <p class="second-p">
+              <a class="strikethrough" href="#">$299</a> $25
+            </p>
+          </div>
+          <div>
+            <img src="./images/suit.jpg" alt="image" />
+            <p>
+              <a href="#"
+                >This Mars helicopter will be the first ever to fly on another
+                planet</a
+              >
+            </p>
+            <p class="second-p">
+              <a class="strikethrough" href="#">$299</a> $25
+            </p>
+          </div>
+          <div>
+            <img src="./images/suit.jpg" alt="image" />
+            <p>
+              <a href="#"
+                >This Mars helicopter will be the first ever to fly on another
+                planet</a
+              >
+            </p>
+            <p class="second-p">
+              <a class="strikethrough" href="#">$299</a> $25
+            </p>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="ftr1 sectionflex">
+        <div class="links">
+          <a href="#"><i class="fab fa-facebook fa-2x"></i></a>
+          <a href="#"><i class="fab fa-twitter fa-2x"></i></a>
```

Comment:
#### Mobile Screen Size

- [ ] The bottom of the footer needs to be different

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91437361-28e70080-e840-11ea-8ec5-e5775b938d70.png)|![image](https://user-images.githubusercontent.com/61479648/91437388-31d7d200-e840-11ea-84d0-49fe1e4abe5d.png) |